client/src/views/Environment.tsx

```diff
@@ -19,51 +20,69 @@ const ENDPOINTS_POLLING_INTERVAL = 5000;
 
 function Environment() {
   const theme = useTheme();
-  const { output, environment, stopEnvironment, relaunchEnvironment } = useOkteto();
+  const { output, environment, status, stopEnvironment, currentContext, relaunchEnvironment } = useOkteto();
   const [endpoints, setEndpoints] = useState<Array<string>>([]);
+  const prevStatusRef = useRef<OktetoStatus | null>(null);
+
 
   const handleOpenEnvironment = () => {
     if (environment) {
       window.ddClient.host.openExternal(environment.link);
     }
   };
 
+  useEffect(() => {
+    if ( prevStatusRef.current && prevStatusRef.current === 'activating' && status === 'synchronizing')
+      window.ddClient.desktopUI.toast.success('env ready to use');
```

Comment:
The text should read like in the UI mock:

```suggestion
      window.ddClient.desktopUI.toast.success('Environment successfully launched!');
```